### ISSUE TITLE
"axis" in Softmax changes by the description method in Keras.

### ISSUE THREAD

**Issue by teeppp** [onnx/keras-onnx#18]
I found the strange behavior when convert the keras model which includes softmax layer to onnx.
The axis in Softmax is different evenif same model, only difference is how to set activation in keras. 
According to [Keras Documentation](https://keras.io/activations/), there are 2 ways to set activation function as follows.
### 2 lines description
```py
model.add(Dense(64))
model.add(Activation('softmax'))
```
or
###  1 line description 
```py
model.add(Dense(64, activation='softmax'))
```
These should be same model. But, conversion results are different as follows.
- "2 lines description" -> axis is set to **-1**.
- "1 lines description" -> axis is set to **1**.

I guess that this is due to how to call "apply_softmax" function.
When we use "2lines description" model, it seems apply_softmax is called in "activation.py".
In this case, arguments for axis is "-1" in "activation.py".
But, when use we use "1 line description" model, apply_softmax seems that it is called by "dense.py" 
In this case, there is no arguments for axis. then, the parameter will use Defalut value(axis=1) in apply_softmax.

I'm not sure which is correct value, but, this should be same.

**Comment by jiafatom** [comment]
checked in [PR](https://github.com/onnx/keras-onnx/pull/20) which set axis=-1 in apply_softmax (default for keras). This should resolve the issue.

**Comment by teeppp** [comment]
@jiafatom 
Thanks for your update. I understood that 'axis = -1' is expected default value.
And, in this case, I think that it is necessary to modify in [conv.py l.170-l.172](https://github.com/onnx/keras-onnx/blob/master/keras2onnx/ke2onnx/conv.py) too.

Could you check if you need to modify the conv.py or not?

By the way, is there any reason to set `axis = 1` as default in apply_softmax in [onnx_ops.py](https://github.com/onnx/keras-onnx/blob/da7ea5a168b2478adb9ff81b1ca31de09795eeee/keras2onnx/common/onnx_ops.py)(l.408)?
If my understanding is correct, you can modify only this file.

**Comment by jiafatom** [comment]
Just checked in [code](https://github.com/onnx/keras-onnx/pull/37) to modify conv.py (set axis=-1). 
We should not change onnx_ops.py since this is for all convertors and onnx spec set axis=1 as default.
